config/manifests/bases/nvidia-network-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: ""
+    capabilities: Basic Install
+    description: Deploy and manage NVIDIA networking resources in Kubernetes
+    provider: NVIDIA
+    repository: https://github.com/Mellanox/network-operator/
+    support: NVIDIA
+  name: nvidia-network-operator.v0.0.0
```

Comment:
Why do we have v0.0.0 here? is it a template?

Reply:
Yes, this is the template